kieserver/modules/kieserver/module.yaml

```diff
@@ -36,6 +36,9 @@ artifacts:
 - name: "slf4j-simple.jar"
   # slf4j-simple-1.7.22.redhat-2.jar
   md5: "62cc6eeb72e2738e3acc8957ca95f37b"
+- name: "jbpm-event-emitters-kafka-7.x.Final.jar"
+  # jbpm-event-emitters-kafka-7.52.0.Final-redhat-00004.jar
```

Comment:
wouldn't be better to use there the name with version? @spolti 
Also aren't there needed some extra changes for future (auto) updates - new versions?

Reply:
@almope is this jar built on brew?
can we get it using brew calls?

Reply:
@desmax74 remember to update the cacher as well to update this jar when pushing automated PRs.
https://github.com/kiegroup/kie-cloud-tools/tree/master/cekit-helpers

Reply:
ok